prboom2/src/m_menu.c

```diff
@@ -3199,10 +3198,11 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Game speed, percentage of normal", S_NUM|S_PRGWARN, m_null, G_X, G_Y+13*8, {"realtic_clock_rate"}},
   {"Default skill level",              S_CHOICE,        m_null, G_X, G_Y+14*8, {"default_skill"}, 0, 0, NULL, gen_skillstrings},
   {"Default compatibility level",      S_CHOICE,        m_null, G_X, G_Y+15*8, {"default_compatibility_level"}, 0, 0, NULL, &gen_compstrings[1]},
-  {"Show ENDOOM screen",               S_YESNO,         m_null, G_X, G_Y+16*8, {"showendoom"}},
-  {"Fullscreen menu background",       S_YESNO, m_null, G_X, G_Y + 17*8, {"menu_background"}},
+  {"Show Secret Notifications",        S_YESNO,         m_null, G_X, G_Y+16*8, {"hudadd_secretarea"}},
```

Comment:
Maybe that's at least one thing we can get out of this PR after all. Does "Show Secret Notifications" make more sense to a native speaker than "REPORT REVEALED SECRETS"? Is it immediately clear what is meant? I am open to rename the menu item once again if there is consensus about an improved wording.

Reply:
I mean, the notifications are not secret, they are about secrets. But maybe that's just me?

Reply:
No, I agree that it sounds much better to me as well.

Reply:
BTW what about my other proposal, of moving the automap settings to a new page under General, or renaming Setup to something more meaningful?

Reply:
If we agree to overhaul the menu, we should do it as a whole and not move a single section. Once you _know_ where to find a setting, it doesn't matter that much anymore how hard it was to find it. What we should probably avoid is to move some settings to a new logical place but leave others where they were. I am open to renaming some of the sections, though I think that "Setup" is quite fitting for settings things like the display resolution.

Reply:
Yes, Setup in itself is not a poor name, the issue is what it currently houses. Stuff like HUD and automap settings is the more glaring issue in my book, instead of housing resolution, FOV, compatibility, the ENDOOM screen, and the list goes on.

Honestly I'd be just fine with an overhaul, but I'm not even sure where to begin with something so grand in scale. Any suggestions?